railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= camelized %></title>
+    <title><%= content_for(:title) || camelized %></title>
```

Comment:
I wonder if we should handle I18n by default in these cases. Something like [this gem](https://github.com/calebhearth/title), or even `t("title")`

Reply:
I used this gem before until I noticed that most of my projects start without internationalization.

Reply:
I think Rails could be more proactive (since it's already shipping with the I18n gem) and help in this case. Not sure what the solution would look like, but I'd love to see something on that direction.

Reply:
https://twitter.com/dhh/status/962131305344335872 😁

Reply:
If English-only is enough for BC, good for them. But [Rails is a big tent](https://rubyonrails.org/doctrine#big-tent) (even that page has multiple languages).

I think we can move forward for the better, and supporting I18n for titles would set the example (like we did when we included the `test` dir by default).

Reply:
Yeah, I understand, if it would be a more sophisticated helper it could use an internationalization structure.